cost/google/cud_expiration/Google_CUD_Expiration_Report.pt

```diff
@@ -1,9 +1,8 @@
-name "Google Committed Use Discount (CUD) Expiration Report"
+name "Google Committed Use Discount (CUD) Expiration"
```

Comment:
The terminology here that sounds better to me is "Google Expiring Commi....". Generally we have the policies named with what they're checking for. But not sure about consistency with other ri expiration policies. This is acceptable

Reply:
Thats good feedback.  I made those changes and added the Azure Expiring Reserved Instance to the Root readme.  Can I get one more review to merge to master?